docs/sources/administration/data-source-management/team_ldap.md

```diff
@@ -0,0 +1,130 @@
+  - ../permissions/datasource_permissions/team_ldap
+  - ../permissions/datasource_permissions/teamldap
+description: Data source permissions for Teams for Grafana administrators
+labels:
+  products:
+    - enterprise
+    - cloud
+title: Data source permissions for Team LDAP
+weight: 100
+---
+
+# Team LBAC
+
+Grafana's new Team LBAC (Label-Based Access Control) feature for Loki is a significant enhancement that simplifies and streamlines data source access management based on team memberships. Users wanting fine grained access to data sources such as Loki with X amount of teams with different levels of access can make use of Team LBAC.
+
+This feature addresses a common challenge faced by Grafana users: managing Loki data source access for different teams. Previously, this led to the creation of numerous connections and duplicate dashboards, hampering scalability and user experience. With Team LBAC, users can now configure access to specific labels based solely on team memberships.
+
+## Team LBAC rules
+
+Users who want teams with a specific set of label selectors can add rules for each team. Team LBAC rules have no upper limit as to how many you can configure per team.
+
+Team LBAC rules are added to the http request to Loki data source. Setting up Team LBAC rules for any team will apply those rules to the teams.
+
+**Note:** Any teams within Grafana without a rule will be able to query all logs if there are role based queriying setup. See <> for more information.
+
+Configuring multiple rules for a team acts the same way as CAP rules, each rule is evaluated separately. If a team has two rules configured for it, both rules will be applied to the request and the result will be the [logical disjunction](https://en.wikipedia.org/wiki/Logical_disjunction), a "OR" operation of the two rules.
+
+Only a data source administrator with permission access can edit LBAC rules at the data source permissions tab. Changing LBAC rules requires the same access level as editing data source permissions (admin permission for data source).
+
+> "Can I use CAPs (cloud access policies) together with TeamLBAC rules?"
+No, CAP always have precedence. If there are any CAP configured for the same datasource and there are TeamLBAC rules configured, then only the CAP access will be applied.
+
+> "If administrator forget to add rule for a team, what happens?"
+The teams that does not have a rule applied to it, would be able to query the logs if `query` permissions are setup for their role within Grafana.
+
+#### Best practices
+For configuring new rules, or viewing the rules for the teams we recommend testing the rules in the Loki Explore view. This will allow you to see the logs that would be returned for the rule.
+
+
+#### Scenarios
+**Scenario 1: One rule setup for a Team**
+We have two teams, Team A and Team B. Loki access is setup with `Editor`, `Viewer` roles to have `Query` permission.
+
+Team A has a rule `namespace="auth"` configured. 
```

Comment:
I would suggest something like `namespace="dev"` instead. `auth` sounds pretty much like a secured stuff, so it might add some confusion.